test/modules/logicModule/RecurringPaymentManager.t.sol

```diff
@@ -318,7 +337,11 @@ contract RecurringPaymentManagerTest is ModuleTest {
 
     function testRemoveRecurringPaymentModifierInPosition() public {
         //Init Module
-        recurringPaymentManager.init(_proposal, _METADATA, abi.encode(1 weeks));
+        recurringPaymentManager.init(
+            _proposal,
+            _METADATA,
+            abi.encode(1 weeks, hasDependency, dependencies)
```

Comment:
These encodings still contain hasDependency although that should have been moved over to the proposalfactory
I think this happened on multiple places in the code

Reply:
Yes, true. Have fixed this.